Primo.js

```diff
@@ -265,8 +278,10 @@ function importPNX(text) {
 	}
 	
 	item.title = ZU.xpathText(doc, '//display/title');
-	if(item.title) item.title = ZU.unescapeHTML(item.title);
-	
+	if(item.title) {
+		item.title = ZU.unescapeHTML(item.title);
+	    item.title = item.title.replace(/\s*:/, ":");
```

Comment:
we already do this for import via MARC and from WorldCat, so this should be entirely uncontroversial.